### PR TITLE
Visualize losses produced by an aristotle calculation

### PR DESCRIPTION
**Pull request by ptormene**
image

The outputs in aristotle mode are displayed using a new template that we can further customize.
For now, all outputs and all export formats are available, and the losses table is presented on top (see the screenshot).
Below the table I also added a button to download the table as csv.

TODO:
* give a better format to numbers and perhaps add the dollar sign before economic losses
* add timestamp and magnitude of the event

Part of #9227

### CODE REVIEW

File: openquake/commands/plot_assets.py [hunk 1]
```diff
@@ -98,7 +98,21 @@ def main(calc_id: int = -1, site_model=False, shapefile_path=None,
     w, h = maxx - minx, maxy - miny
     ax.set_xlim(minx - 0.2 * w, maxx + 0.2 * w)
     ax.set_ylim(miny - 0.2 * h, maxy + 0.2 * h)
-    ax.legend()
+
+    try:
+        iso_codes = [
+            iso_code.decode('utf8')
+            for iso_code in
+            dstore['assetcol/tagcol/ID_0'][numpy.unique(assetcol['ID_0'])]]
```
Comment (micheles) [inline openquake/commands/plot_assets.py:106]: For performance reasons, the right way to do this is as follows:
```python
ALL_ID_0 =  dstore['assetcol/tagcol/ID_0'][:]
ID_0 = ALL_ID_0[numpy.unique(assetcol['ID_0'])]
```

File: openquake/server/views.py [hunk 2]
```diff
@@ -1264,6 +1263,75 @@ def web_engine_get_outputs_aelo(request, calc_id, **kwargs):
                        warnings=warnings))
 
 
+def get_aggrisk(calc_id):
```
Comment (micheles) [inline openquake/server/views.py:1266]: The docstring should say something like "Converting the aggrisk dataframe into a table like this:
..." and add an example table.

File: openquake/server/views.py [hunk 3]
```diff
@@ -1264,6 +1263,75 @@ def web_engine_get_outputs_aelo(request, calc_id, **kwargs):
...
+        return HttpResponseNotFound()
+    if not utils.user_has_permission(request, job.user_name):
+        return HttpResponseForbidden()
+    body, header = get_aggrisk(calc_id)
+    # Create the HttpResponse object with the appropriate CSV header.
+    response = HttpResponse(
+        content_type="text/csv",
+        headers={
+            "Content-Disposition":
+                'attachment; filename="aggrisk.csv"'
```
Comment (micheles) [inline openquake/server/views.py:1325]: Please check what happens when downloading the other aggrisk output (i.e. is the name "aggrisk.csv" already taken or not?)

File: openquake/commands/plot_assets.py (hunk 1 above)
Comment (ptormene) [reply openquake/commands/plot_assets.py]: Done

File: openquake/server/views.py (hunk 2 above)
Comment (ptormene) [reply openquake/server/views.py:1266]: Done

File: openquake/server/views.py (hunk 3 above)
Comment (ptormene) [reply openquake/server/views.py:1332]: The name of the csv file created downloading the standard aggrisk outputs is something like output-101-aggrisk-_45.csv, I suppose we might use a similar approach here, adding the calc_id, and we might decide to further differentiate the name.

Comment (ptormene) [reply openquake/server/views.py:1332]: We should add the calc_id

Comment (ptormene) [reply openquake/server/views.py]: Done

File: openquake/commands/plot_assets.py [hunk 4]
```diff
@@ -98,7 +98,20 @@ def main(calc_id: int = -1, site_model=False, shapefile_path=None,
...
     ax.set_ylim(miny - 0.2 * h, maxy + 0.2 * h)
-    ax.legend()
+
+    try:
+        ALL_ID_0 = dstore['assetcol/tagcol/ID_0'][:]
+        ID_0 = ALL_ID_0[numpy.unique(assetcol['ID_0'])]
+    except KeyError:  # ID_0 might be missing
+        ax.legend()
+    else:
+        id_0_str = ', '.join([id_0.decode('utf8') for id_0 in ID_0])
```
Comment (micheles) [inline openquake/commands/plot_assets.py:108]: The square brackets are not needed

**Review by micheles — APPROVED** [review]

File: openquake/commands/plot_assets.py (hunk 4 above)
Comment (ptormene) [reply openquake/commands/plot_assets.py]: Removed

**Pull request merged**